### ISSUE TITLE
A11y_VS Code: Copilot_Suggestion_ScreenReader : Screen reader is not announcing tooltip information when we press CTRL+K+I.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode#188252]
#A11yTCS; #A11ySev2; #DesktopApp; #ScreenReader; #WCAG1.3.1; #Win11; #Win32; #NVDA; #Info and Relationships; #A11yWCAG2.1; #SH-VS Code Copilot-Win32-July23; #Visual Studio Code Client; #Regressed:08-11-23;

### Actual Result:​
Screen reader is not announcing tooltip information when we press CTRL+K+I.

### Expected Result:​
Screen reader should announce the tooltip information when we press CTRL+K+I.

### Environment Details:
Application Name: VS Code: Copilot
Visual studio code Insiders Version: 1.81.0(User Setup)
Microsoft Windows 11 Enterprise 22H2 Build 22621.1992
NVDA Version: 2022.2.3
Jaws

### Repro Steps:

1. Open the "Visual Studio Code" dialog box.
2. In Visual Studio Code, create a new JavaScript (*.js) file.
3. In the code Paste this code function calculateDaysBetweenDates(begin, end) { and press enter.
4. A suggestion will appear.
5. Prees "Ctrl + K, Ctrl + I" and a tooltip will appear.
6. Verify whether the screen reader is announcing tooltip information or not.

### User Impact:
Screen reader users will not get information available on tooltip.

### Attachment: 

NVDA: 
![Bug_188252_NVDA](https://github.com/microsoft/vscode/assets/124244478/62160f8d-4cee-4214-9b28-702234b5817d)
JAWS: 
![Bug_188252_Jaws](https://github.com/microsoft/vscode/assets/124244478/f8281e86-b9dd-4b11-8911-08eeeadf2153)

**Comment by AnKushSingh05** [comment]
@meganrogge Can you Please reopen the bug, this bug will only be close once verified. Thanks!

**Comment by meganrogge** [comment]
maybe related #https://github.com/microsoft/vscode/issues/190261 or https://github.com/microsoft/vscode/pull/190209

**Comment by meganrogge** [comment]
duplicate of https://github.com/microsoft/vscode/issues/190261 and will be fixed in the next insider's

**Comment by AnKushSingh05** [comment]
Regress the issue in version: 1.82.0-insider (user setup).
Issue is fixed. Hence closing the bug.
PFA for the reference.

![Bug_188252_Fixed](https://github.com/microsoft/vscode/assets/124244478/52c8777e-d6a4-47ff-8b3a-b925050fb004)
![Bug_188252_Fixed_Jaws](https://github.com/microsoft/vscode/assets/124244478/9dee7e8c-1bcd-4074-936f-42a00ecad7f9)
